### PR TITLE
Create termedia-neuropsychiatria-i-neuropsychologia-neuropsychiatry-a…

### PR DESCRIPTION
**Pull request by martynaut**
…nd-neuropsychology.csl

Termedia Neuropsychiatria i neuropsychologia (neuropsychiatry and neuropsychology) journal style file

in this PR new style file was created for polish Neuropsychiatria i neuropsychologia (Neuropsychiatry and Neuropsychology) journal from Termedia publisher.

### CODE REVIEW

**Comment by martynaut** [comment]
Why there is a rendering failure for sample citations? Does it mean it has to be changed in any way? The style works for me in zotero.

**Comment by adam3smith** [comment]
I'm seeing both Polish and English articles in the journal -- I think it'd make sense to remove the default locale entirely so both are possible? What do you think?

File: termedia-neuropsychiatria-i-neuropsychologia-neuropsychiatry-and-neuropsychology.csl [hunk 1]
```diff
@@ -94,12 +98,6 @@
       </if>
     </choose>
   </macro>
-  <locale>
-    <terms>
-      <term name="page-range-delimiter">-</term>
-      <term name="et-al"> i wsp.</term>
```
Comment (martynaut) [inline termedia-neuropsychiatria-i-neuropsychologia-neuropsychiatry-and-neuropsychology.csl]: If we remove language PL how this one line will work? This should apply to PL language only. I'm not super fluent in CSL for zotero, sorry if this one is stupid question :).
The other option to removing language PL is preparing two versions of styles one for PL articles and other one for ENG.

Comment (martynaut) [reply termedia-neuropsychiatria-i-neuropsychologia-neuropsychiatry-and-neuropsychology.csl]: @adam3smith

File: termedia-neuropsychiatria-i-neuropsychologia-neuropsychiatry-and-neuropsychology.csl [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    <eissn>2084-9885</eissn>
+    <updated>2020-12-17T09:00:00+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="page-range-delimiter">-</term>
+    </terms>
+  </locale>
+  <locale xml:lang="pl">
```
Comment (adam3smith) [inline termedia-neuropsychiatria-i-neuropsychologia-neuropsychiatry-and-neuropsychology.csl:26]: @martynaut You can specify localized terms per language like this

**Comment by adam3smith** [comment]
This looks good to me -- @martynaut if you could look over/test this one more time and give your OK, we can merge

**Comment by martynaut** [comment]
@adam3smith tested, looks ok, thank you for your help!

**Comment by adam3smith** [comment]
Thanks!

**Pull request merged**